pkg/auth/server/login/login.go

```diff
@@ -164,19 +165,26 @@ func (l *Login) handleLogin(w http.ResponseWriter, req *http.Request) {
 		failed(errorCodeUserRequired, w, req)
 		return
 	}
+	var result string
+	defer func() {
+		metrics.Record(result)
+	}()
 	user, ok, err := l.auth.AuthenticatePassword(username, password)
 	if err != nil {
 		glog.Errorf(`Error authenticating %q with provider %q: %v`, username, l.provider, err)
```

Comment:
Note, this should be utilruntime.HandleError(...), not glog.Errorf().

Reply:
I opened a separate PR for that.